pkg/amutex/amutex_test.go

```diff
@@ -47,7 +47,7 @@ func TestMutualExclusion(t *testing.T) {
 	// goroutines ran concurrently within the critical section.
 	//
 	// If one of the goroutines doesn't complete, it's likely a bug that
-	// causes to to wait forever.
+	// causes to wait forever.
```

Comment:
I think this should be "caused it to wait forever".

Reply:
Done.